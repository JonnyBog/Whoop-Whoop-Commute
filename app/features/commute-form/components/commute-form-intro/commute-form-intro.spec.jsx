import React from 'react';
import renderer from 'react-test-renderer';

import CommuteFormIntro from 'features/commute-form/components/commute-form-intro/commute-form-intro';

describe('Features', () => {
  describe('Commute Form Intro Component', () => {
    it('should export', () => {
      expect(CommuteFormIntro).toBeDefined();
    });

    /**
     * createRenderTree
     * @returns {JSX} component tree
     */
    function createRenderTree () {
      return renderer.create(
        <CommuteFormIntro />
      ).toJSON();
    }

    describe('Appearance', () => {
      it('should render', () => {
        const tree = createRenderTree();
        expect(tree).toMatchSnapshot();
      });
    });
  });
});
