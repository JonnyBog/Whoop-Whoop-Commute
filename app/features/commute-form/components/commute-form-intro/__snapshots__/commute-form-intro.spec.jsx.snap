// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Features Commute Form Intro Component Appearance should render 1`] = `
<p
  className="commute-form-intro__CommuteFormIntroCopy-n5yvyc-0 bOrnia"
>
  Pick a work station and an area to travel from on the map - then see the different commutes from that area!
</p>
`;
